daemon/volumes_linux.go

```diff
@@ -36,12 +36,13 @@ func (container *Container) setupMounts() ([]execdriver.Mount, error) {
 		if err != nil {
 			return nil, err
 		}
-
-		mounts = append(mounts, execdriver.Mount{
-			Source:      path,
-			Destination: m.Destination,
-			Writable:    m.RW,
-		})
+		if container.trySetNetworkMount(m.Destination, path) == false {
```

Comment:
I will changes this. 
And since it seem it will make make it 'in', I'll add test cases as well. Thanks